cmd/destroy/remote.go

```diff
@@ -93,7 +106,7 @@ func newRemoteDestroyer(manifest *model.Manifest) *remoteDestroyCommand {
 	builder := remoteBuild.NewBuilderFromScratch()
 	return &remoteDestroyCommand{
 		builder:              builder,
-		destroyImage:         manifest.Destroy.Image,
+		destroyImage:         "",
```

Comment:
if we set this always to `""` we are not allowing to set a destroy image in manifest. A solution would be something like: 
`````go

func newRemoteDestroyer(manifest *model.Manifest) *remoteDestroyCommand {
	fs := afero.NewOsFs()
	builder := remoteBuild.NewBuilderFromScratch()
	if manifest.Destroy == nil {
		manifest.Destroy = &model.DestroyInfo{}
	}
	return &remoteDestroyCommand{
		builder:              builder,
		destroyImage:         manifest.Destroy.Image,
		fs:                   fs,
		workingDirectoryCtrl: filesystem.NewOsWorkingDirectoryCtrl(),
		temporalCtrl:         filesystem.NewTemporalDirectoryCtrl(fs),
		manifest:             manifest,
		registry:             builder.Registry,
		clusterMetadata:      fetchClusterMetadata,
	}
}
`````

Reply:
Good one! I have applied those changes here.

I think one way of doing the same another way would have been initializing the DestroInfo at the time when the manifest has been found but it'd consist of more code. This works best.